dev/PushNotifications/PushNotifications-Constants.h

```diff
@@ -7,12 +7,19 @@
 #define _STRINGIZE(x)   _STRINGIZEX(x)
 #endif
 
+#define _WSTRINGIZEXX(x) L ## #x
+#define _WSTRINGIZEX(x) _WSTRINGIZEXX(x)
+#define _WSTRINGIZE(x) _WSTRINGIZEX({ ## x ## })
```

Comment:
Why this?

Why does this differ from _STRINGIZE[X] above? That comes from yvals_core.h (as noted), and only of necessity.

Reply:
WideString vs String. That's the difference. It was needed for CLSIDFromString API usage.